src/libsyntax/feature_gate.rs

```diff
@@ -689,6 +686,8 @@ declare_features! (
     (accepted, repr_packed, "1.33.0", Some(33158), None),
     // Allows calling `const unsafe fn` inside `unsafe` blocks in `const fn` functions.
     (accepted, min_const_unsafe_fn, "1.33.0", Some(55607), None),
+    // `extern crate self as foo;` puts local crate root into extern prelude under name `foo`.
+    (accepted, extern_crate_self, "1.33.0", Some(56409), None),
```

Comment:
```suggestion
    (accepted, extern_crate_self, "1.34.0", Some(56409), None),
```
This won't make it to 1.33.